src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -89,6 +114,102 @@ export const HookPasswordValidatedInput = ({
   );
 };
 
+export const ValidatedInputAndTextArea = ({
+  value,
+  stepValidation,
+  fieldName,
+  onBlur,
+  placeholder,
+  onChange,
+  ariaLabel,
+  inputType = 'textInput',
+}: ValidationInputProp) => {
+  const errorMessage = stepValidation.errors[fieldName];
+  const hasError = errorMessage !== '';
+
+  return (
+    <>
+      <TextAndTextAreaInput
+        inputType={inputType}
+        ariaLabel={ariaLabel || ''}
+        value={value}
+        placeholder={placeholder || ''}
+        errorMessage={errorMessage}
+        onChange={onChange!}
+        onBlur={onBlur!}
```

Comment:
Why do we need a non-null assertion operator here?

Reply:
same